css/properties/print-color-adjust.json

```diff
@@ -1,55 +1,67 @@
 {
   "css": {
     "properties": {
-      "color-adjust": {
+      "print-color-adjust": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/color-adjust",
-          "spec_url": "https://drafts.csswg.org/css-color-adjust/#perf",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/print-color-adjust",
+          "spec_url": "https://drafts.csswg.org/css-color-adjust/#propdef-print-color-adjust",
           "support": {
             "chrome": {
               "version_added": "49",
-              "alternative_name": "-webkit-print-color-adjust"
+              "prefix": "-webkit-"
             },
             "chrome_android": {
               "version_added": "49",
-              "alternative_name": "-webkit-print-color-adjust"
+              "prefix": "-webkit-"
             },
             "edge": {
               "version_added": "79",
-              "alternative_name": "-webkit-print-color-adjust"
-            },
-            "firefox": {
-              "version_added": "48"
-            },
-            "firefox_android": {
-              "version_added": "48"
+              "prefix": "-webkit-"
             },
+            "firefox": [
+              {
+                "version_added": "97"
+              },
+              {
+                "version_added": "48",
+                "alternative_name": "color-adjust"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "97"
+              },
+              {
+                "version_added": "48",
+                "alternative_name": "color-adjust"
+              }
+            ],
             "ie": {
               "version_added": false
             },
             "opera": {
               "version_added": "15",
-              "alternative_name": "-webkit-print-color-adjust"
+              "prefix": "-webkit-"
             },
             "opera_android": {
               "version_added": "36",
-              "alternative_name": "-webkit-print-color-adjust"
+              "prefix": "-webkit-"
             },
             "safari": {
               "version_added": "6",
-              "alternative_name": "-webkit-print-color-adjust"
+              "prefix": "-webkit-"
             },
```

Comment:
Fwiw, Safari added the unprefixed version in STP: https://webkit.org/blog/12193/release-notes-for-safari-technology-preview-139/

Reply:
Thanks @nt1m !  

@ddbeck would right way to do this be to modify with `preview` as shown below? 

```
"safari": [
  {
     "version_added": "preview"
   },
  {
     "version_added": "6",
     "version_removed": "preview"
     "prefix": "-webkit-"
   }
],
"safari_ios": [
  {
     "version_added": "preview"
   },
  {
     "version_added": "6",
     "version_removed": "preview"
     "prefix": "-webkit-"
   }
],
```

Reply:
Yes, except I don't see any indication that the `-webkit-` prefix has been removed. I'd remove the `version_removed` lines.

Reply:
Great, done except that I used Safari 15.4 to match the iOS entry, and https://developer.apple.com/documentation/safari-release-notes/safari-15_4-release-notes